REQUIREMENTS.md

```diff
@@ -0,0 +1,160 @@
+# IPLD Requirements
+
+This document outlines parts of IPLD that should and should not be changed to
+ensure the success of future improvements (especially type systems).
+
+**Definitions**
+
+* Block: A block is a chunk of an IPLD DAG, encoded in a format. Blocks have CIDs.
+* Fragment: A piece of an IPLD DAG. Blocks contain fragments.
```

Comment:
I don't think we've had much cause to use this term recently.  Node and Block come up constantly.  Fragment hasn't... at all, really, in my working memory; or if it does, it's been colloquial and not in a way that really seems to deserve a glossary entry.

Reply:
Possibly. IIRC, I defined it because I needed it but I probably just ended up removing its use.